src/erasure-code/jerasure/ErasureCodeJerasure.cc

```diff
@@ -158,10 +158,21 @@ bool ErasureCodeJerasure::is_prime(int value)
     151,157,163,167,173,179,
     181,191,193,197,199,211,223,227,229,233,239,241,251,257
   };
-  int i;
-  for (i = 0; i < 55; i++)
-    if (value == prime55[i])
+  int i = sizeof(prime55)/sizeof(int) -1;
```

Comment:
but i don't think `is_prime()` is the concern of performance here, we only check for prime when we are parsing a profile, and normally the number `w` being checked is very small, so i think `std::find()` is more appropriate and faster here in most cases.